spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONArray.java

```diff
@@ -39,7 +39,7 @@
  * In particular, {@code get} fails if the requested index holds the null reference, but
  * succeeds if it holds {@code JSONObject.NULL}.
  * <p>
- * Instances of this class are not thread safe. Although this class is nonfinal, it was
+ * Instances of this class are not thread safe. Although this class is non-final, it was
```

Comment:
This class is "shaded" from third-party source and should be left as-is.

Reply:
Done.